### ISSUE TITLE
NameError: name 'lists' is not defined

### ISSUE THREAD

**Issue by alexmaday** [thatguywiththatname/PCPartPicker-API#1]
Hi Simon, cool idea for this API! Glad I stumbled on your project while I was trying to get at the great data on PCPart Picker for an excel project so I can quote PC builds etc.

Running the example code after installing your API via pip, the `NameError` for the 'lists' variable came up. It seems a small re-factoring will fix that right up. I'd be happy to do a PR, (it'd be my first, so I may need guidance) or you can check it out on your own.
Here's the exact output:

```
pcpp.productLists.totalPages('cpu')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\alexm\AppData\Local\Programs\Python\Python37-32\lib\site-packages\PCPartPicker_API\pcpartpicker.py", line 45, in totalPages
    return lists._getPage(partType, 1, True)
NameError: name 'lists' is not defined
```

Cheers

**Comment by thatguywiththatname** [comment]
Hi,

Thanks for letting me know about this - its happening because I recently changed the `lists` class to a (hopefully) easier to understand `productLists` class.

I thought I had cleaned everything up but apparently I missed that!

I'm happy for you to PR the fix, and if you find any other problems, want to improve my code, or even add a new feature, feel free to open a PR!

**Comment by thatguywiththatname** [comment]
In smaller projects like this I would generally just apply the fix on the main branch, but if you want to do it that way, sure.

**Comment by thatguywiththatname** [comment]
Closed by #2
